storage/SystemInfo/CmdBlkid.h

```diff
@@ -103,6 +105,8 @@ namespace storage
 	bool any_luks() const;
 	bool any_bcache() const;
 
+        static list<string> split_line( const string & line );
```

Comment:
In general vector is better than list. Since makeMap is only used in CmdBlkid that could be change now. Clearly optional.